include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1426,27 +1921,48 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::submit_gradient(
 
 
 template <int n_components, int dim, int spacedim, typename Number>
-inline DerivativeForm<1, dim, spacedim>
+inline DerivativeForm<1, dim, spacedim, Number>
 FEPointEvaluation<n_components, dim, spacedim, Number>::jacobian(
   const unsigned int point_index) const
 {
   const auto &mapping_data =
     mapping_info->get_mapping_data(current_cell_index, current_face_number);
-  AssertIndexRange(point_index, mapping_data.jacobians.size());
-  return mapping_data.jacobians[point_index];
+
+  const unsigned int n_lanes = internal::VectorizedArrayTrait<Number>::width;
+  DerivativeForm<1, dim, spacedim, Number> vectorized_jacobian;
+  for (unsigned int v = 0;
+       v < n_lanes && point_index * n_lanes + v < mapping_data.jacobians.size();
+       ++v)
+    for (unsigned int d = 0; d < dim; ++d)
+      for (unsigned int s = 0; s < spacedim; ++s)
+        internal::VectorizedArrayTrait<Number>::get(vectorized_jacobian[d][s],
+                                                    v) =
+          mapping_data.jacobians[point_index * n_lanes + v][d][s];
+  return vectorized_jacobian;
 }
 
 
 
 template <int n_components, int dim, int spacedim, typename Number>
-inline DerivativeForm<1, spacedim, dim>
+inline DerivativeForm<1, spacedim, dim, Number>
```

Comment:
For completeness: These functions now return `DerivativeForm<1, spacedim, dim, float>` instead of `DerivativeForm<1, spacedim, dim, double (default)>` if `Number=float`.